### PR TITLE
454 improve quick start

### LINKED ISSUE #395: few minor changes in docs
**1. Please add something like: change namespace in Lens to observe the cluster starting up, otherwise nothing will be displayed in Lens**

> image

______________________________________________________________
**2. Add a description how one can get the developement_values.yaml file via wget to simplify the process**

> image

______________________________________________________________
**3. Please change to "an edge device"**

> image

### LINKED ISSUE #454: Improve Quick Start Guide for mechanical engineers and for programmers
**Is your feature request related to a problem? Please describe.**

Currently, it is unintuitive for mechanical engineers how to setup the system. Programmers can stuck as well.

**Describe the solution you'd like**

The Quick Start should include two options:
1. Quick setup using k3OS and a cloudconfig (e.g., umh.app/examples/brewing.yaml)
2. Quick setup using minikube

### LINKED ISSUE #472: Add to docs: "Common IT tools and techniques"
Add to docs: "Common IT tools and techniques" and link it in the entire documentation. Examples: 

Link in doc (e.g., Quick Start): Connect via SSH to the Karbon 300.

In the article: What is SSH? WHat tools are out there? MobaXTerm. Any additional reads.

Let's do that with SSH, git, Deploying a Kubernetes Cluster, etc.

The goal is to make it as easy as possible for mechanical engineers.

### PR DESCRIPTION
**Pull request by JeremyTheocharis**
FIxes #454
Fixes #395 
Fixes #472

### CODE REVIEW

**Comment by JeremyTheocharis** [comment]
https://deploy-preview-499--umh-docs.netlify.app/docs/getting-started/setup-development/

**Comment by Gornoka** [comment]
> https://deploy-preview-499--umh-docs.netlify.app/docs/getting-started/setup-development/

thanks a lot

File: docs/content/en/docs/Getting started/setup-development/index.md [hunk 1]
```diff
@@ -1,90 +1,172 @@
...
-## Steps
+### Prerequisites
 
-### k3OS
+- a edge device with x86 architecture. We recommend using the [K300 from OnLogic](https://www.onlogic.com/eu-en/k300/)
+- the [latest version of k3OS](https://github.com/rancher/k3os/releases/) [^versioning] installed on a bootable USB-stick [^flash-usb]. 
+- a computer with SSH / SFTP client [^SSH-client] and [Lens](https://k8slens.dev/) (for accessing the Kubernetes cluster) installed. We recommend a laptop with an Ethernet port or with an Ethernet adapter. 
+- local LAN (with DHCP) available via atleast two Ethernet cables and access to the internet.[^network-setup].
+- a computer monitor connected with the edge device 
+- a keyboard connected with the edge device
```
Comment (larswitte) [inline docs/content/en/docs/Getting started/setup-development/index.md:27]: Great, maybe add a graphic, would make it even easier.

File: docs/content/en/docs/Getting started/setup-development/index.md [hunk 2]
```diff
@@ -1,90 +1,172 @@
...
-mqttBridgePrivkey: |
-  ENTER CERT HERE
-```
+[^boot-usb]: See also out guide: [How to boot from a USB-stick](/docs/getting-started/understanding-the-technologies/#installing-operating-systems-from-a-usb-stick)
 
-6. Execute `helm install factorycube-edge /home/rancher/united-manufacturing-hub/deployment/factorycube-edge --values "/home/rancher/united-manufacturing-hub/deployment/factorycube-edge/development_values.yaml" --set serialNumber=$(hostname) --kubeconfig /etc/rancher/k3s/k3s.yaml  -n factorycube-edge` (change kubeconfig and serialNumber accordingly) (Please pay attention to the correct path. The path may be /home/rancher/united-manufacturing-hub-main ... or similar)
+#### Getting access to the device
 
-### Install factorycube-server
+To verify whether the installation worked and access [Grafana] (the dashboard) and [Node-RED], we will first enable SSH via password authentification, fetch the login details for [Kubernetes] and then login via [Lens].
```
Comment (larswitte) [inline docs/content/en/docs/Getting started/setup-development/index.md:51]: Insert "to":
To verify whether the installation worked and **to** access [Grafana] (the dashboard) and [Node-RED], we will first enable SSH via password authentification, fetch the login details for [Kubernetes] and then login via [Lens].

File: docs/content/en/docs/Getting started/setup-development/index.md [hunk 3]
```diff
@@ -1,90 +1,172 @@
...
+
+Enable SSH password authentification in k3OS [^ssh-password-authentication]. This is currently not necessary anymore as the automated setup script will do that automatically, **therefore this step can be skipped**. This paragraph only exists to remind you that this setting is not the default behavior of k3OS and should be deactivated in production environments.
+
+For production environments we recommend using a certificate to authenticate, which is enabled by default. This can be archieved by modifying the cloud-init file and [linking to a public key stored on your GitHub account.](https://gist.github.com/dmangiarelli/1a0ae107aaa5c478c51e#ssh-setup-with-putty)
+
+[^ssh-password-authentication]: See also out guide: [Enabling k3os password authentication](/docs/tutorials/add-username-password-authentification-k3os-ssh/)
+
+#### Step 3: Connect via SSH
+
+Connect via SSH [^SSH-client] from your laptop with the edge device. The IP address is shown on the computer screen on your edge device (see also step 1). If it is not available anymore, you can view the current IP address using `ip addr`.
```
Comment (larswitte) [inline docs/content/en/docs/Getting started/setup-development/index.md:86]: Username is "rancher"

File: docs/content/en/docs/Getting started/setup-development/index.md [hunk 4]
```diff
@@ -1,90 +1,172 @@
...
+
+#### Step 4: Getting Kubernetes credentials
+
+Execute `cat /etc/rancher/k3s/k3s.yaml` in your SSH session on your laptop  to retrieve the Kubernetes credentials. Copy the content of the result into your clipboard.
+
+{{< imgproc k3s_secret_1.png Fit "1280x500" >}}Execute `cat /etc/rancher/k3s/k3s.yaml`{{< /imgproc >}}
+
+{{< imgproc k3s_secret_2.png Fit "1280x500" >}}Copy the content{{< /imgproc >}}
+
+Connect with the edge device and the Kubernetes credentials in your clipboard.
```
Comment (larswitte) [inline docs/content/en/docs/Getting started/setup-development/index.md:96]: Not easily understandable. Instead maybe something like:
"Connect the program Lens with the edge device using the Kubernetes credentials from your clipboard."

File: docs/content/en/docs/Tutorials/add-username-password-authentification-k3os-ssh/index.md [hunk 5]
```diff
@@ -5,6 +5,8 @@ description: >
   This article explains how to enable the classic username / password authentification for SSH in k3os
```
Comment (larswitte) [inline docs/content/en/docs/Tutorials/add-username-password-authentification-k3os-ssh/index.md:5]: Why would you want to add SSH password authentification: what actually changes?
And what are the dangers?

File: docs/content/en/docs/Tutorials/install-k3os/index.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+---
+title: "How to install k3OS"
```
Comment (larswitte) [inline docs/content/en/docs/Tutorials/install-k3os/index.md:2]: Great! You can link it from the "0. Understanding the technologies"/"Installing operating systems from a USB-stick" chapter.

**Review by larswitte — APPROVED** [review]
Great improvement @JeremyTheocharis!

Maybe you could also add some kind of overview site, structuring and linking to all your documentation articles. This might improve navigation significantly.

**Review by Scarjit — APPROVED** [review]
Upon merging, a new Issue should be created to resolve the remaining #TODO's

File: docs/content/en/docs/Getting started/understanding-the-technologies/index.md [hunk 7]
```diff
@@ -3,90 +3,368 @@ title: "0. Understanding the technologies"
...
+
+#### Architecture
+
+*See also [Architecture](/docs/concepts/)*
+
+#### Example projects
+
+*See also [Examples](/docs/examples/)*
+
+## Deep-dive: IT technologies and techniques
```
Comment (larswitte) [inline docs/content/en/docs/Getting started/understanding-the-technologies/index.md:286]: I really like the content but I feel like **Introduction into IT/OT** and **Deep-dive: IT technologies and techniques** don't really belong logically together. I would maybe either split it up into two files under the **Getting Started** Category or I would move the content of **Deep-dive: IT technologies and techniques** into the **Tutorial** Category.

**Review by larswitte — APPROVED** [review]
Really nice, I found one thing I would improve, but its probably just a matter of taste.

File: docs/content/en/docs/Getting started/understanding-the-technologies/index.md (hunk 7 above)
Comment (JeremyTheocharis) [reply docs/content/en/docs/Getting started/understanding-the-technologies/index.md:286]: We're my thoughts as well. I'll do it tomorrow.

**Review by Scarjit — APPROVED** [review]

File: docs/content/en/docs/Tutorials/add-username-password-authentification-k3os-ssh/index.md [hunk 8]
```diff
@@ -5,14 +5,20 @@ description: >
...
 
+**DANGER: NOT RECOMMENDED FOR PRODUCTION! USE DEFAULT BEHAVIOR WITH CERTIFICATES INSTEAD**
+
+By default, k3OS allows SSH connections only using certificates. This is a much safer method than using passwords. However, we realized that most mechanical engineers and programmers are overwhelmed with the creation of a [public key infrastructure](/docs/tutorials/pki/). Therefore, it might make sense to enable password authentication in k3OS for development mode.
+
 ## Prerequisites
 
 - Edge device running [k3OS](https://github.com/rancher/k3os)
-- SSH access to that device
+- physical access to that device
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/add-username-password-authentification-k3os-ssh/index.md:15]: physical -> Physical

**Review by Scarjit — APPROVED** [review]

**Pull request merged**